xml/System.Reflection/Assembly.xml

```diff
@@ -4041,7 +4041,11 @@ This property is marked obsolete starting in .NET 5, and generates a compile-tim
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- <xref:System.IO.FileLoadException> is thrown if `assemblyRef` specifies the full assembly name and the first assembly that matches the simple name has a different version, culture, or public key token. The loader does not continue probing for other assemblies that match the simple name. Starting with .NET Framework 4, because execution of code in remote assemblies is disabled by default, a <xref:System.IO.FileLoadException> is also thrown if `assemblyRef` specifies a remote assembly. To enable execution of code loaded from remote locations, you can use the [`<loadFromRemoteSources>`](/dotnet/framework/configure-apps/file-schema/runtime/loadfromremotesources-element) configuration element.
+ <xref:System.IO.FileLoadException> is thrown if `assemblyRef` specifies the full assembly name and the first assembly that matches the simple name has a different version, culture, or public key token. The loader does not continue probing for other assemblies that match the simple name.
+
+  **.NET Framework only:** Starting with .NET Framework 4, because execution of code in remote assemblies is disabled by default, a <xref:System.IO.FileLoadException> is also thrown if `assemblyRef` specifies a remote assembly. To enable execution of code loaded from remote locations, you can use the [`<loadFromRemoteSources>`](/dotnet/framework/configure-apps/file-schema/runtime/loadfromremotesources-element) configuration element.
```

Comment:
This is the behavior that .NET Framework 4.0 shipped with, but it was later revisited in .NET Framework 4.5 since it was super annoying - it made programs crash in unexpected ways when running from remote shares.
 
For example, I just tried to run this on .NET Framework on corpnet and it works fine:
```
using System;
using System.Reflection;

class Program
{
    static void Main()
    {
        // Load random managed assembly from \\clrmain
        Assembly a = Assembly.LoadFrom(@"\\clrmain\tools\managed\HyperAddin\bin\Debug\HyperAddin.dll");
        Console.WriteLine(a);
    }
}
```
According to this doc, it should fail since `\\clrmain\tools\...` is a remote share, but it does not in practice.

I think we should delete the discussion of remote assemblies from these docs. It is no longer relevant, and the text is not accurate either. The accurate documentation is at https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/loadfromremotesources-element . It is very long and complicated story.

We can just link to that doc from here if we wanted to. Something like:

**.NET Framework only**: See https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/loadfromremotesources-element for loading assemblies from remote locations.